src/tests/functional/plugin/conformance/subgraphs_dumper/tests/matchers/subgraph/read_value.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tuple>
+
+#include "gtest/gtest.h"
+
+#include "matchers/subgraph/read_value_assign.hpp"
+#include "utils/model.hpp"
+#include "utils/model_comparator.hpp"
+
+#include "test_models/model_0.hpp"
+#include "test_models/model_4.hpp"
+#include "test_models/model_5.hpp"
+#include "base_test.hpp"
+
+#include "openvino/pass/manager.hpp"
+#include "openvino/pass/serialize.hpp"
+
+namespace {
+
+using namespace ov::tools::subgraph_dumper;
+
+
+// ======================= ExtractorsManagerTest Unit tests =======================
+class ReadValueAssignExtractorTest : public SubgraphsDumperBaseTest {
+protected:
+    ReadValueAssignExtractor extractor = ReadValueAssignExtractor();
+
+    bool is_match(const std::vector<ReadValueAssignExtractor::ExtractedPattern>& models,
```

Comment:
We have the same class per every extractor test, correct? To separate the general logic to base class?

Reply:
done